src/__tests__/__snapshots__/RootReducer.test.ts.snap

```diff
@@ -16,6 +16,7 @@ Object {
   },
   "actionQueue": Object {
     "actionQueueMap": Object {},
+    "activeProgramIds": Array [],
```

Comment:
General comment for this commit: Did you test starting a new program after the last one errored and remained in the store?

Reply:
Yes, this behavior has been observed. That's the only way to show the error is if the program state remains in the redux store.